docs/src/main/sphinx/connector/bigquery.md

```diff
@@ -74,14 +74,41 @@ bigquery.project-id=<your Google Cloud Platform project id>
 
 ### Multiple GCP projects
 
-The BigQuery connector can only access a single GCP project.Thus, if you have
+The BigQuery connector can only access a single GCP project. Thus, if you have
 data in multiple GCP projects, You need to create several catalogs, each
 pointing to a different GCP project. For example, if you have two GCP projects,
 one for the sales and one for analytics, you can create two properties files in
 `etc/catalog` named `sales.properties` and `analytics.properties`, both
 having `connector.name=bigquery` but with different `project-id`. This will
 create the two catalogs, `sales` and `analytics` respectively.
 
+### Understanding Project ID Resolution
+
+The BigQuery connector determines the project ID to use based on the 
```

Comment:
Somewhere we need to link to some sort docs from BigQuery that explains more about the project ID ideally .. maybe right here

Reply:
Link the `project ID` to the link I found